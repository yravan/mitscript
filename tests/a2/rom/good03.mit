dict = {};
if (dict.a == None) {
  print("First we access");
}
print(dict);
print("If you got here, then there aren't segfaults due to map access without key!");
