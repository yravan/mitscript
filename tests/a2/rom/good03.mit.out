First we access
{}
If you got here, then there aren't segfaults due to map access without key!
